ios/engine/KMEI/KeymanEngine/Classes/Resource Management/ResourceDownloadQueue.swift

```diff
@@ -416,13 +416,17 @@ class ResourceDownloadQueue: HTTPDownloadDelegate {
           let keyboards = batch.resources as! [InstallableKeyboard]
           log.info("Downloaded keyboard: \(keyboards[0].id).")
 
+          // TEMP:  wrap the newly-downloaded resources with a kmp.json.
+          //        Serves as a bridge until we're downloading actual .kmps for keyboards.
+          let wrappedKeyboards = Migrations.migrateToKMPFormat(keyboards)
```

Comment:
This helps provide a decent stopping place for this PR's changes.  We instantly migrate any newly-downloaded Cloud-JS keyboards; in a later PR, we'll swap to downloading KMPs instead.